Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
@alyssais I created ```Library/Homebrew/rubocops.rb```, and it has a single line ```Dir[File.dirname(__FILE__) + "/rubocops/*.rb"].each {|file| require file }```

Line 10 in ```Library/.rubocop.yml``` is now ``` - ./Homebrew/rubocops.rb```

Now just run ```brew audit --strict <someformula>``` to see the error.
Further debugging shows the error is thrown at line 72 in ```/Homebrew/cmd/style.rb```
and I think the issue is being unable to ```require``` ```bottle_block_cop.rb```